app/utils/kcidb.py

```diff
@@ -103,3 +143,58 @@ def push_build(build_id, first, bq_options, db_options={}, db=None):
     bq_data['builds'] = [bq_build]
 
     _submit(bq_data, bq_options)
+
+
+def push_tests(group_id, bq_options, db_options={}, db=None):
+    if db is None:
+        db = utils.db.get_db_connection(db_options)
+    collection = db[models.TEST_GROUP_COLLECTION]
+    group = utils.db.find_one2(collection, group_id)
+    origin = bq_options.get("origin", "kernelci")
+    ns = bq_options.get("namespace", "kernelci.org")
+    test_cases = _get_test_cases(group, db, [], ns)
+    build = _get_build_doc(group, db)
+    build_id = _make_id(build[models.ID_KEY], ns)
+    env_description = "{} in {}".format(
+        group[models.DEVICE_TYPE_KEY],
+        group[models.LAB_NAME_KEY]
+    )
+    test_description = "{} on {} in {}".format(
+        group[models.NAME_KEY],
+        group[models.DEVICE_TYPE_KEY],
+        group[models.LAB_NAME_KEY]
+    )
+    misc = {
+        'plan': group[models.NAME_KEY],
```

Comment:
BTW I think we should change that to something a bit more obvious than `plan` and `plan_variant`, but that's the best we can do with what is currently in the backend.